samples/ibm-samples/watson/watson_train_serve_pipeline.py

```diff
@@ -16,60 +16,65 @@
 
 # generate default secret name
 import os
-secret_name = 'ai-pipeline-' + os.path.splitext(os.path.basename(CONFIG_FILE_URL))[0]
+import kfp
+from kfp import components
+from kfp import dsl
+import ai_pipeline_params as params
 
+secret_name = 'kfp-creds'
+configuration_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/commons/config/component.yaml')
+train_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/watson/train/component.yaml')
+store_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/watson/store/component.yaml')
+deploy_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/watson/deploy/component.yaml')
+    
 # create pipelines
-import kfp.dsl as dsl
-import ai_pipeline_params as params
 
 @dsl.pipeline(
     name='KFP on WML training',
     description='Kubeflow pipelines running on WML performing tensorflow image recognition.'
 )
 def kfp_wml_pipeline(
+    GITHUB_TOKEN=dsl.PipelineParam(name='github-token',
+                                   value=''),
+    CONFIG_FILE_URL=dsl.PipelineParam(name='config-file-url',
+                                      value='https://raw.githubusercontent.com/user/repository/branch/creds.ini'),
+    train_code=dsl.PipelineParam(name='train-code', value='tf-model.zip'),
+    execution_command=dsl.PipelineParam(name='execution-command', value='\'python3 convolutional_network.py --trainImagesFile ${DATA_DIR}/train-images-idx3-ubyte.gz --trainLabelsFile ${DATA_DIR}/train-labels-idx1-ubyte.gz --testImagesFile ${DATA_DIR}/t10k-images-idx3-ubyte.gz --testLabelsFile ${DATA_DIR}/t10k-labels-idx1-ubyte.gz --learningRate 0.001 --trainingIters 20000\''),
+    framework= dsl.PipelineParam(name='framework', value='tensorflow'),
+    framework_version = dsl.PipelineParam(name='framework-version', value='1.5'),
+    runtime = dsl.PipelineParam(name='runtime', value='python'),
+    runtime_version = dsl.PipelineParam(name='runtime-version', value='3.5'),
+    run_definition = dsl.PipelineParam(name='run-definition', value='wml-tensorflow-definition'),
+    run_name = dsl.PipelineParam(name='run-name', value='wml-tensorflow-run'),
+    model_name=dsl.PipelineParam(name='model-name', value='wml-tensorflow-mnist'),
```

Comment:
You can do this without explicit `PipelineParam`s:
```suggestion
    model_name='wml-tensorflow-mnist',
```

Reply:
thanks - fixed